contracts/VotingMachines/GenesisProtocol.sol

```diff
@@ -77,7 +77,7 @@ contract GenesisProtocol is IntVoteInterface,UniversalScheme {
     mapping(bytes32=>Parameters) public parameters;  // A mapping from hashes to parameters
     mapping(bytes32=>Proposal) public proposals; // Mapping from the ID of the proposal to the proposal itself.
 
-    uint constant public NUM_OF_CHOICES = 2;
+    uint constant public MAX_NUM_OF_CHOICES = 2;
```

Comment:
GenesisProtocol  currently support exactly 2 num of choices ..so MAX is not appropriate here.

Reply:
Perhaps then we can add MIN_NUM_OF_CHOICES too?

I need a consistent voting machine interface to be able to validate in Arc.js that a user's voting choices are within a range allowable by the voting machine.  It needs to do this in a voting machine-agnostic way.

Thus I am also proposing that MAX_NUM_OF_CHOICES (and MIN_NUM_OF_CHOICES) be made part of `IntVoteMachine` (see #444).

Reply:
how about in arc.j : if GP so num of choices  =2 ?

Reply:
> how about in arc.js : if GP so num of choices =2 ?

That doesn't work in a voting machine-agnostic context.  Refer to my comment just above.

Reply:
already addressed at #454